specification/resources/resource-manager/Microsoft.Resources/stable/2019-03-01/resources.json

```diff
@@ -2,7 +2,7 @@
   "swagger": "2.0",
   "info": {
     "title": "ResourceManagementClient",
-    "version": "2018-05-01",
```

Comment:
So we had this API version? That's bad. I assume, now we may have some SDK breaking changes.

Reply:
we haven't released any package with this swagger, so I think it's still fine.

Reply:
@Tiano2017 as far as I know, if the spec was merged to master, some SDK packages could be released.

Reply:
Not sure what's the right approach here. This change is more like fixing a bug, and I assume it's only going to impact future versions of packages. can we assume it's safe in that case?